meilisync/schemas.py

```diff
@@ -15,10 +17,14 @@ class Event(ProgressEvent):
     data: dict
 
     def mapping_data(self, fields_mapping: Optional[dict] = None):
-        if not fields_mapping:
-            return self.data
         data = {}
         for k, v in self.data.items():
-            if k in fields_mapping:
-                data[fields_mapping[k] or k] = v
+            if isinstance(v, datetime.datetime):
+                v = int(v.timestamp())
```

Comment:
It is because that's the proper way to use dates in Meilisearch, when we index using timestamps its possible to use filters/sorting the way we expect :)

You can read more here: https://www.meilisearch.com/docs/learn/advanced/working_with_dates